.github/workflows/test-action.yml

```
@@ -13,4 +13,4 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Execute maintenance
-        uses: ./ # Uses an action in the root directory
\ No newline at end of file
+        uses: ./ # Uses an action in the root directory
```

actionlint (1.7.7)

missing input "github_token" which is required by action "GitHub Maintenance Action" defined at "./". all required inputs are "github_token"

(action)